src/components/EnterpriseApp/data/hooks/useEnterpriseCuration.js

```diff
@@ -73,6 +85,11 @@ function useEnterpriseCuration({ enterpriseId, curationTitleForCreation }) {
         if (!curation) {
           curation = await createEnterpriseCuration();
         }
+        const curationHighlightedContents = await Promise.all(curation.highlightSets.map(async (highlightSet) => {
+          const data = await getHighlightSet(highlightSet.uuid).then(error => error);
```

Comment:
What does `.then(error => error)` do here?

Reply:
good catch. I removed the error handling here since the error should be caught in the catch block instead